src/model/data.types.model.ts

```diff
@@ -16,3 +16,21 @@ export enum BeneficialOwnerTypeChoice {
     otherLegal = "otherLegal",
     none = "none"
 }
+
+
+export interface InputDate {
+    day: number
+    month: number
+    year: number
+}
+
+export enum natureOfControl {
+    over25under50Percent = "25",
```

Comment:
Thanks for adding the 'percent'.

Looked at your screen and it matches mine. Options are:

over 25% up to (and including) 50%
more than 50% and less than 75%
75% or more
In order for this to be reusable and more accurate, I suggest renaming the first and last of these to:

over25upTo50Percent and 75PercentOrMore.

Reply:
"atLeast75Percent" would be better I don't really want to start the name with a number.

Reply:
Sounds good.